cli_tools/gce_vm_image_import/importer/importer.go

```diff
@@ -240,7 +223,7 @@ func Run(clientID string, imageName string, dataDisk bool, osID string, customTr
 		description, *region, subnet, network, noGuestEnvironment, sysprepWindows)
 
 	var w *daisy.Workflow
-	if w, err = runImport(ctx, varMap, importWorkflowPath, zone, timeout, *project, scratchBucketGcsPath,
+	if w, err = runImport(context.Background(), varMap, importWorkflowPath, zone, timeout, *project, scratchBucketGcsPath,
```

Comment:
if object ctx is not shared across functions, it's not necessary to pass it into runImport. Can call context.Background() inside runImport

Reply:
sgtm, done